dgbpy/dgbtorch.py

```diff
@@ -268,8 +269,19 @@ def get_criterion( info, params ):
     return globals()[criterion]()
   raise ValueError('Unsupported loss function: %s' % criterion)
 
+def _load_torch_pt_model( modelfnm ):
+  try:
+    model = torch.jit.load( modelfnm )
+  except RuntimeError:
+    raise RuntimeError('Unsupported model, only torch scripted models are supported')
+  return model
+
 def load( modelfnm ):
   model = None
+  if dgbhdf5.isTorchModelFile( modelfnm ) and not dgbhdf5.isHDF5File( modelfnm ):
```

Comment:
This does not belong here. modelfnm is exclusively the hdf5 file, thus never the model implementation for the PyTorch platform

The function _load_torch_pt_model may be exported, it does not need to stay internal.